example/policies/incident.md

```diff
@@ -104,7 +104,7 @@ a. The ISM must take all necessary steps to resolve the incident and recover inf
 
     i. Disclosures (parties to which the details of this incident were disclosed to, such as customers, vendors, law enforcement, etc.)
 
-a. After an incident has been resolved, the ISM must conduct a post mortem that includes root cause analysis and documentation any lessons learned.
+a. After an incident has been resolved, the ISM must conduct a postmortem that includes root cause analysis and documentation any lessons learned.
```

Comment:
This is an amusing one. The Latin is two words, and Google says that it's still written more often as two words, but dictionaries say that the word is preferred as one word.